migtests/tests/mysql/datatypes/validate

```diff
@@ -143,13 +144,20 @@ def migration_completed_checks(tgt):
 	for table_name, row_count in EXPECTED_ROW_COUNT.items():
 		print(f"table_name: {table_name}, row_count: {got_row_count[table_name]}")
 		assert row_count == got_row_count[table_name]
+	
+	EPSILON = 1e-9  # Define a small tolerance for floating-point comparisons
```

Comment:
pls add a comment here explaining why we did this (i.e. 2.25). Link to GH ticket, if any.